pkg/fedora/tarsnap.spec

```diff
@@ -0,0 +1,59 @@
+Name:           tarsnap
+Version:        1.0.36.1
+Release:        2%{?dist}
+Summary:        Secure, efficient online backup service
+
+License:        Proprietary
+URL:            http://www.tarsnap.com/
+Source0:        http://www.tarsnap.com/download/tarsnap-autoconf-%{version}.tgz
+
+BuildRequires:  gcc
+BuildRequires:  pkgconfig(bash-completion)
+BuildRequires:  pkgconfig(bzip2)
```

Comment:
Are you sure we need `bzip2`?  I think we need `zlib-devel`, but not `bzip2`.

Also, on CentOS 7-1.1503.el7 (I hope this is "similar enough" to Fedora, but please let me know if it isn't), trying to install the build dependencies:

```
sudo yum install yum-utils
sudo yum-builddep pkg/fedora/tarsnap.spec
```

gives me:

```
Getting requirements for pkg/fedora/tarsnap.spec
 --> Already installed : gcc-4.8.3-9.el7.x86_64
 --> Already installed : 1:bash-completion-2.1-6.el7.noarch
 --> Already installed : e2fsprogs-devel-1.42.9-7.el7.x86_64
 --> Already installed : 1:openssl-devel-1.0.1e-42.el7.9.x86_64
 --> xz-devel-5.1.2-12alpha.el7.x86_64
 --> Already installed : zlib-devel-1.2.7-13.el7.x86_64
 --> libacl-devel-2.2.51-12.el7.x86_64
 --> libattr-devel-2.4.46-12.el7.x86_64
Error: No Package found for pkgconfig(bzip2)
```

Could you comment on writing
```
BuildRequires:  pkgconfig(bzip2)
```
instead of
```
BuildRequires:  bzip2
```
?  if I remove the `pkgconfig()` part, it installs `bzip2` without problem.